### PR TITLE
MINOR: Add Mysql Lineage & Usage Support 1.5.12

### PR DESCRIPTION
**Pull request by ulixius9**
### Describe your changes:

Fixes <issue-number>

I worked on ... because ...

#
### Type of change:

- [ ] Bug fix
- [ ] Improvement
- [ ] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] My PR title is `Fixes <issue-number>: <short explanation>`
- [ ] I have commented on my code, particularly in hard-to-understand areas. 
- [ ] For JSON Schema changes: I updated the migration scripts or explained why it is not needed.

### CODE REVIEW

File: openmetadata-spec/src/main/resources/json/schema/entity/services/connections/database/mysqlConnection.json [hunk 1]
```diff
@@ -95,6 +95,9 @@
       "title": "Supports Profiler",
       "$ref": "../connectionBasicType.json#/definitions/supportsProfiler"
     },
+    "supportsLineageExtraction": {
+      "$ref": "../connectionBasicType.json#/definitions/supportsLineageExtraction"
+    },
```
Comment (ayush-shah) [inline openmetadata-spec/src/main/resources/json/schema/entity/services/connections/database/mysqlConnection.json:100]: do we require supportsUsageExtraction here as well?

**Review by harshach — APPROVED** [review]

File: ingestion/src/metadata/ingestion/source/database/mysql/queries.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+	NULL `database_name`,
+	argument `query_text`,
+	event_time `start_time`,
+    NULL `end_time`,
+	NULL `duration`,
+	NULL `schema_name`,
+    NULL `query_type`,
+    NULL `user_name`,
+    NULL `aborted`
+FROM mysql.general_log
```
Comment (pmbrull) [inline ingestion/src/metadata/ingestion/source/database/mysql/queries.py:29]: I think we need to add a line on the docs about enabling these events right

Comment (ulixius9) [reply ingestion/src/metadata/ingestion/source/database/mysql/queries.py:29]: yes I'll raise a different pr to add it in main and add docs

Comment (ulixius9) [reply ingestion/src/metadata/ingestion/source/database/mysql/queries.py:29]: oh you meant the side panel, I'll add it there

**Review by ayush-shah — APPROVED** [review]

**Pull request merged**